src/layouts/MenuLayout/components/MenuPanel/components/Action.js

```diff
@@ -0,0 +1,37 @@
+// @flow strict
+
+import React from 'react'
+
+import {
+  JIcon,
+  JLink,
+} from 'components/base'
+
+import menuPanelStyle from '../menuPanel.m.scss'
+
+type Props = {|
+  +href: string,
+  +label: string,
+  +iconName: string,
+|}
+
+export function Action({
+  href,
+  label,
+  iconName,
+}: Props) {
+  return (
+    <li>
```

Comment:
If we will use this component only in context `Actions` component, it's fine.
Otherwise, will better, if we don't make guess-work about context where component will be put.